### PR TITLE
ApheaBio.RcloneTray version 1.0.0

### PR DESCRIPTION
**Pull request by aurang1655**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/17690)

### CODE REVIEW

**Review by aurang1655 — COMMENTED** [review]
Manifest file oke

**Review by ghost — COMMENTED** [review]
Maybe add  params for silent install?

**Review by ghost — APPROVED** [review]
Everything seems oke

File: manifests/a/ApheaBio/RcloneTray/1.0.0/ApheaBio.RcloneTray.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: ApheaBio.RcloneTray
+PackageVersion: 1.0.0
+Installers:
+- Architecture: x86
+  InstallerType: nullsoft
+  InstallerUrl: https://github.com/aurang1655/ApheaBio/releases/download/1/RcloneTray_dist_RcloneTray_Setup_1.0.0.exe
```
Comment (ItzLevvie) [inline manifests/a/ApheaBio/RcloneTray/1.0.0/ApheaBio.RcloneTray.installer.yaml:9]:
```
  InstallerUrl: https://github.com/aurang1655/ApheaBio/releases/download/1/RcloneTray_dist_RcloneTray_Setup_1.0.0.exe
```
Shouldn't the URL be https://github.com/dimitrov-adrian/RcloneTray/releases/download/v1.0.0/rclonetray-setup-1.0.0.exe here? With a hash of `b86f8b6123ce540934c89e1f629101ee04bae088c10f6e999fc58f97a1d2faeb`?

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Pull request closed without merging**